configs/components/openssl.rb

```diff
@@ -126,8 +126,6 @@
       no-weak-ssl-ciphers \
       no-dtls \
       no-dtls1 \
-      no-rc4 \
```

Comment:
Can we update the line above that states:

> # Below will disable: bf, camellia, des, 3des, gost, idea, md2/4, rc2/4/5, seed, srp

I believe DES is still on, and this PR re-renables RC4